Rakefile

```diff
@@ -1,5 +1,6 @@
+require 'puppet_litmus/rake_tasks' if Bundler.rubygems.find_name('puppet_litmus').any?
+require 'puppetlabs_spec_helper/rake_tasks'
```

Comment:
What tasks are we using from PSH?

Reply:
Litmus runs `spec_prep` which comes from PSH

Reply:
TODO: Can I just import spec_prep?

Reply:
It looks like maybe `require 'puppetlabs_spec_helper/tasks/fixtures'` would maybe pick up just `spec_prep`, `spec_clean`, and `spec_clean_symlinks`. Not sure if it depends on stuff rom the parent file though.

Reply:
Looks good. Will try.

Reply:
Took your advice and fixed.